.github/workflows/release-connect-init.yml

```diff
@@ -0,0 +1,92 @@
+name: "[Release] connect NPM and v9"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit_sha:
+        description: "The commit SHA to checkout"
+        required: true
+        type: string
+      deploymentType:
+        description: "Select the deployment type. (example: canary, stable)"
+        required: true
+        type: choice
+        options:
+          - canary
+          - stable
+
+jobs:
+  # Version should have been bumped by now thanks to ./ci/scripts/connect-release-init-npm.js
+  extract-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.set-version.outputs.version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Number of commits to fetch. 0 indicates all history for all branches and tags.
+          fetch-depth: 0
+          # Checkout the specified commit
+          ref: ${{ github.event.inputs.commit_sha }}
+
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Extract connect version
+        id: set-version
+        run: echo "version=$(node ./ci/scripts/get-connect-version.js)" >> $GITHUB_OUTPUT
+
+  create-push-release-branch:
+    needs: [extract-version]
+    name: "Create release branch for version ${{ needs.extract-version.outputs.version }}"
+    runs-on: ubuntu-latest
+    outputs:
+      branch_name: ${{ steps.push-branch.outputs.branch_name }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          # Ensure the full commit history is available is required to get specific `ref`.
+          fetch-depth: 0
+          # Checkout the specified commit
+          ref: ${{ github.event.inputs.commit_sha }}
+
+      - name: Create and push new branch
+        env:
+          BRANCH_NAME: "release/connect/${{ needs.extract-version.outputs.version }}"
+        run: |
+          echo ${{ env.BRANCH_NAME }}
+          git checkout -b ${{ env.BRANCH_NAME }}
+          git push origin ${{ env.BRANCH_NAME }}
```

Comment:
@vdovhanych as agreed I am not adding here the "trezor-ci" bot credential and name since this branch is going to be created by the user that triggers the action.

Reply:
How is this supposed to work? Its trying to push te code to protected branch. It wont work like this, we don't want to have automation rights to push commits to protected branch.